src/components/tooltip/TooltipDirective.js

```diff
@@ -118,7 +118,10 @@ goog.require('ga_topic_service');
                 updateCursorStyle, 10, false, false);
 
         // Register click/touch/mousemove events on map
-        var registerMapEvents = function(scope, onClick) {
+        var registerMapEvents = function(scope, onClick, deregEvent) {
```

Comment:
But it's 2 different calls. I tried anyway and we end up with the same issue. Without an instance or variable outside the function, it won't work.
If you call:
https://github.com/geoadmin/mf-geoadmin3/blob/master/src/components/tooltip/TooltipDirective.js#L283
and then
https://github.com/geoadmin/mf-geoadmin3/blob/master/src/components/tooltip/TooltipDirective.js#L270
dereg will always be equal to `angular.noop`.
I understant it's better to be self contained though.